contracts/coordinator/src/state.rs

```diff
@@ -228,6 +228,16 @@ pub fn deployed_contracts(
         .ok_or(report!(Error::DeploymentNameNotFound(deployment_name)))
 }
 
+pub fn deployments(storage: &dyn Storage) -> Result<Vec<ChainContracts>, Error> {
```

Comment:
this could potentially be a whole lot. It's dangerous to just return a "table" wholesale. If this grows to a size that costs too much gas to fully process, you're going to brick every contract that is dependent on this call. We have other examples in the code base where we use pagination to get around that issue